CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ project(KinesisVideoProducerC LANGUAGES C)
 option(ADD_MUCLIBC "Add -muclibc c flag" OFF)
 option(BUILD_DEPENDENCIES "Whether or not to build depending libraries from source" ON)
 option(USE_OPENSSL "Use openssl as crypto library" ON)
+option(USE_MBEDTLS "Use openssl as crypto library" OFF)
```

Comment:
use mbedTLS?

Reply:
all our variables are capitalized though.

Reply:
+1 - capitalized is used everywhere